docs/connectors/orchestration.md

```diff
@@ -69,13 +69,13 @@ Running           Running            Yes
 
 3. Start connector `c1`:
    ```
-   fda table-connector example numbers c1 start
+   fda connector example numbers c1 start
    ```
    Now the Changes Stream tab will show new input records from both connectors.
 
 4. Pause connector `c2`:
    ```
-   fda table-connector example numbers c2 pause
+   fda connector example numbers c2 pause
```

Comment:
Could you also change the `curl` example of getting connector stats to use FDA as well (I had to use curl because fda support wasn't available)?